tests/createWcsWithSip.py

```diff
@@ -79,6 +78,7 @@ def testBigXy0(self):
             src.set(yKey, src.get(yKey) - 500 + y0)
         bbox = afwGeom.Box2I(afwGeom.Point2I(x0, y0), afwGeom.Extent2I(1000, 1000))
         res = self.astrom.determineWcs2(cat, bbox=bbox)
+        self.assertTrue(res.sipWcs is not None, "Failed to fit SIP terms")
```

Comment:
Thanks for the prodding. I've had a look in more detail and in the case where no message is included the explicit test is clearer than `assertTrue`. In the case where a contextual message is included both options result in the same output. I think I'd still prefer the `assertIsNotNone` version if only because it makes it easier for me to find the old-style `assertTrue` code that still exists.